### ISSUE TITLE
Blank Screen

### ISSUE THREAD

**Issue by JBaby9783** [advplyr/audiobookshelf-app#1334]
### What was the Problem?

This blankness has happened twice now on iOS 18. The white screenshot is from Sunday and I was AirPlaying to an HomePod mini. The black screen is now and I was AirPlaying to my Apple TV that has HomePods acting as the TV speakers.
![IMG_0347](https://github.com/user-attachments/assets/c09baf49-5c95-41e8-ba3b-a43155c78248)
![IMG_0349](https://github.com/user-attachments/assets/5de547e0-8ef5-4442-860b-1f7b12e401c7)

### Steps to Reproduce the Issue

Play audiobook for one hour or more. Turn on the phone. Go to the app and it will be blank.

### What was Expected?

When I go back to ABS I should be see the app. There shouldn’t just be nothing.

### Phone Model

iPhone 15 Pro Max

### Phone OS

iOS 18

### Audiobookshelf App Version

iOS App - 0.9.76

### Installation Source

Testflight

**Comment by advplyr** [comment]
Are you able to consistently reproduce this?

**Comment by JBaby9783** [comment]
Yes! In fact I’ve come back here to give an update. I just finished a novella that didn’t take an hour read. I still had the blank screen when I came back to the app. I hadn’t been using my phone because so I was eating so it’s not because of multitasking.

**Comment by advplyr** [comment]
Do you think this is specific to AirPlay? I'm not as familiar with iOS as I am Android so I've never actually AirPlayed anything.

**Comment by JBaby9783** [comment]
I'm not sure. I haven't tested to see if it blanks out while listening via AirPods. I normally only listen with earbuds when I'm away from home.

**Comment by PulsarFTW** [comment]
Just wanted to comment to say it does not seem specifically related to airplay. I have had it with bluetooth headphones today. 

I _think_ I have had it whilst listening to the book directly from the phone speaker, but I do that so rarely that I don't want to confidently say so.

**Comment by mdeeter** [comment]
This is not specific to airplay.  I get this often... I just swtiched from an iPhone 12 pro to an iPhone 16.... that upgrade also included going from iOS 17 to iOS 18.

It was working fine on my iPhone 12 on iOS 17.x

I'm not sure on exact steps to reprodce tho.... at this point I just know:

I listen to ABS on my airpods.
I frequently:
- play games on my phone while ABS plays in the background
- leave ABS playing in the background while my phone is locked
- play ABS while traveling in vehicle with CarPlay

Occasionally, I'll re-open ABS by clicking on the app icon and it will open.... but is a completely black screen.  One time it was black but I could see the play controls.

I'll force close the app and re-open and it works fine.

**Comment by genebean** [comment]
I am actually trying to put ABS on an Android 8.1.0 device, and it installs fine, but when I open the app all I see is a disconnected message on a white background and the ABS logo in the upper left - no menus, not buttons, no form fields, nothing.... it looks very much like this issues screenshot.

**Comment by genebean** [comment]
Here is what I see right after install:

https://github.com/user-attachments/assets/01bf21b7-8aaa-4012-88ec-5ecff10a18f5

**Comment by mdeeter** [comment]
Happened again this morning... I think these are the exact steps that occurred.

Went to bed last night... 
Played book listening on air pods. ..
Opened up a game to play while listening...
Opened ABS! Set chapter point and timer to 60 min...
Locked phone and put on mag safe charger...
Woke up this morning, unlocked phone and clocked on ABS icon. 
Black screen.

**Comment by Unlearned6688** [comment]
I listen to about 8-14 hours per day of podcasts and books with the iOS app. (Yes, for real).

I get this black screen a about twice a day.

My best attempt at memory here:

- it started with iOS 18
- I am sideloading the app. Have used an enterprise certificate and a regular Apple Dev certificate. Black screen for both. (I voluntarily left TestFlight long ago to let someone else have the slot. Can't test that way). I haven't tried or seen the black screen with a free Apple dev account eg using AltServer.
- the black screen seems to be random so far, but appears after listening with my device locked (screen off) for "a while" without breaking- maybe an hour, two hours, hard for me to say.
- It doesn't seem to affect the progress tracking of the app/server. Eg when I get the bug, I close the app, reopen, and my spot is saved.
- I will try to keep a mental note and then a written note next time the black screen pops up

**Comment by genebean** [comment]
On the android device i have, the blank screen started with 0.9.71. Prior to that version, instead of everything being white I have a black background and a hamburger menu, though it will not open a la https://github.com/advplyr/audiobookshelf-app/issues/1369

**Comment by AJolly** [comment]
I run into this issue frequently on android - after listening in the background for a while, switching back to audiobookshelf takes several seconds for the UI to load.  I'd love to debug this better, as it keeps me from using the stock client.  Running on an one plus 11 so the hardware is fast.
